boards/arm/stm32h7/weact-stm32h743/src/weact-stm32h743.h

```diff
@@ -46,6 +49,54 @@
 #  endif
 #endif
 
+/* Can't support USB host or device features if USB OTG FS is not enabled */
+
+#ifndef CONFIG_STM32H7_OTGFS
+#  undef HAVE_USBDEV
```

Comment:
why need define HAVE_XXX maros?

Reply:
if not configured OTG FS then there should be no chance to use it in compile

Reply:
but I think no code reference HAVE_XXXX in nuttx code base.

Reply:
I found code-like in nuttx/boards/arm/stm32h7/linum-stm32h753bi

Reply:
OK